pkg/kubelego/configure.go

```diff
@@ -53,32 +54,37 @@ func (kl *KubeLego) TlsIgnoreDuplicatedSecrets(tlsSlice []kubelego.Tls) []kubele
 }
 
 func (kl *KubeLego) processProvider(ing kubelego.Ingress) (err error) {
-
+	var errs []error
 	for providerName, provider := range kl.legoIngressProvider {
 		err := provider.Reset()
 		if err != nil {
-			provider.Log().Error(err)
-			continue
+			errs = append(errs, err)
```

Comment:
:question:  Should we keep the `continue` ? We previously skipped the logic below when reset fails.